_includes/toc.html

```diff
@@ -84,6 +84,9 @@
 {{ space }}{{ listModifier }} {{ listItemClass }} [{{ heading_body | replace: "|", "\|" }}]({% if include.baseurl %}{{ include.baseurl }}{% endif %}#{{ html_id }}){% if include.anchor_class %}{:.{{ include.anchor_class }}}{% endif %}{% endcapture %}
 {% endfor %}
 
+    - {{ listItemClass }} [Footnotes]({% if include.baseurl %}{{ include.baseurl }}{% endif %}#footnotes){% if include.anchor_class %}{:.{{ include.anchor_class }}}{% endif %}
```

Comment:
This'll show a ToC entry even if the heading doesn't exist, right?

Reply:
Yes. I see no good way to fix this, though.

Reply:
Could you wrap it in a `<div style="display:none;">` and unhide it with the JavaScript that inserts the header?

Reply:
I want to avoid JS at all costs.

Reply:
It's probably worth me making a pre-processing step at some point. This seems like that time.